src/util.cpp

```diff
@@ -859,9 +859,13 @@ bool ArgsManager::ReadConfigStream(std::istream& stream, std::string& error, boo
         }
 
         // Check that the arg is known
-        if (!IsArgKnown(strKey) && !ignore_invalid_keys) {
-            error = strprintf("Invalid configuration value %s", option.first.c_str());
-            return false;
+        if (!IsArgKnown(strKey)) {
+            if (!ignore_invalid_keys) {
+                error = strprintf("Invalid configuration value %s", option.first.c_str());
+                return false;
+            } else {
+                LogPrintf("Ignoring unknown configuration value %s\n", option.first);
```

Comment:
This doesn't work because logging is not setup at this point in startup. The warning does not appear in the debug.log.

Reply:
Hmm, I thought we cached early log messages and printed them afterwards. Any suggestions for how to deal with that?

Reply:
Ah, #13088.